src/styles/topbar.styl

```diff
@@ -7,6 +7,7 @@
     right-spaced(32px)
     left-spaced(32px)
 
-    @media (max-width: (768/basefont)rem)
-        right-spaced(16px)
-        left-spaced(16px)
+@media (max-width: (768/basefont)rem)
+    .fil-content-header
```

Comment:
I wonder if this is necessary now as the `.fil-content-head`, parent of every `.fil-content-header`, is hidden below 768px